src/fromager/sources.py

```diff
@@ -14,7 +14,7 @@
 import requests
 import resolvelib
 from packaging.requirements import Requirement
-from packaging.version import Version
+from packaging.version import InvalidVersion, Version, parse
```

Comment:
`parse` is a _very_ generic name to import like this. :-( 

I'll live with it, but if you have other changes to make, consider renaming it on import or importing the module and accessing the function using `version.parse`.

Reply:
okay, I will change it!